news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2017-11-05
+--- 
+
+![](filepath) BANNER IN PROGRESS
+
+After a long hiatus, Chaos, Voli and I are happy to announce the revival of official beatmapping contests: Welcome to the _osu! Mapping Olympiad!_
```

Comment:
Link users, that you directly mention. Users can change names so userid links to new page would be the way to go. This opener doesn't grab anyone's attention by the way: you usually want to go with something more exciting than "wow this was dead and now it's not dead". Generally it'd be cooler to explain what your thing is in very short and concise manner so that people that aren't interested in the idea can stop reading and those that are intrigued get way more into it. 

e.g.: `If you've been wondering why official mapping contests have been few and far inbetween over the last years and are worriedly wondering when your favourite game mode to map is getting one, worry no further as we present the osu! Mapping Olympiad, a mapping contest across all game modes on a regular schedule! It won't be monthly, but it'll be close to that, run by a small team of community members that are just as pumped for more songs to compete on as you are!` 

or anything, literally anything that sounds less like `hey. this was dead, now it's not. be happy.`

Reply:
User ID: Added.

Catchier Intro: Thanks for the example, but I personally think this is way too wordy for being a catchy introduction.. I will talk with Chaos about it, but for now I think mine is achieving more what I wanted (being short and telling what it's about). He can probably come up with something good, though.